automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/PublishSpec.scala

```diff
@@ -21,51 +21,10 @@ import scala.concurrent.{ExecutionContext, ExecutionContextExecutor}
 class PublishSpec extends FreeSpec with WebBrowserSpec with UserFixtures with WorkspaceFixtures with BillingFixtures with CleanUp with Matchers {
 
   implicit val ec: ExecutionContextExecutor = ExecutionContext.global
-  implicit override val patienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(500, Millis)))
+  implicit override val patienceConfig: PatienceConfig = PatienceConfig(timeout = scaled(Span(10, Seconds)), interval = scaled(Span(500, Millis)))
 
-  val autocompleteTextQueryPrefix: String = "cance"  // partial string of "cancer" to test autocomplete
-  val minNumOfResults: Int = 5
 
   "For a user with publish permissions" - {
-    "an unpublished workspace" - {
```

Comment:
`unpublished workspace` tests are in `UnpublishedWorkspaceSpec` enabled with parallel test execution